acme/acme/standalone_test.py

```diff
@@ -137,12 +136,18 @@ def tearDown(self):
 
     def test_it(self):
         max_attempts = 5
+        started = False
+
         while max_attempts:
             max_attempts -= 1
             try:
                 cert = crypto_util.probe_sni(
                     b'localhost', b'0.0.0.0', self.port)
             except errors.Error:
+                if not started:
```

Comment:
nit: I personally this should be rewritten as:

```
self.assertTrue(max_attempts > 0, "Timeout!")
if started:
    time.sleep(1)
else:
    self.thread.start()
    started = True
```

You could put the assertion in the `if` block if you want, but `continue` isn't necessary here.